src/lib/default-project/project-data.js

```diff
@@ -0,0 +1,150 @@
+import {defineMessages} from 'react-intl';
+
+const messages = defineMessages({
+    backdrop: {
+        defaultMessage: 'backdrop1',
+        description: 'Default name for the first backdrop',
+        id: 'gui.defaultProject.backdrop'
+    },
+    pop: {
+        defaultMessage: 'pop',
+        description: 'Default name for the pop sound',
+        id: 'gui.defaultProject.pop'
+    },
+    sprite: {
+        defaultMessage: 'Sprite1',
+        description: 'Name for the default sprite',
+        id: 'gui.defaultProject.sprite'
+    },
+    costume1: {
+        defaultMessage: 'costume1',
+        description: 'Name for the first default costume',
+        id: 'gui.defaultProject.costume1'
+    },
+    costume2: {
+        defaultMessage: 'costume2',
+        description: 'Name for the second default costume',
+        id: 'gui.defaultProject.costume2'
+    },
+    meow: {
+        defaultMessage: 'Meow',
+        description: 'Name for the meow sound',
+        id: 'gui.defaultProject.meow'
+    },
+    variable: {
+        defaultMessage: 'my variable',
+        description: 'Name for the default variable',
+        id: 'gui.defaultProject.variable'
+    }
+});
+
+// use the default message if a translation function is not passed
+const defaultTranslator = msgObj => msgObj.defaultMessage;
+
+/**
+ * Generate a localized version of the default project
+ * @param {function} translateFunction a function to use for translating the default names
+ * @return {json} the project data json for the default project
```

Comment:
This would be `string`, but it doesn't return a string. But it could! But probably it shouldn't. The `object` it returns is more versatile.